src/components/ImportParameters.jsx

```diff
@@ -0,0 +1,158 @@
+import { useState } from "react";
+
+const parameterDetails = {
+  resource: { 
+    name: <><code>resource</code></>, 
+    type: "IRI, String", 
+    description: <>The file name to write to. 
+      If it contains an extension, this is used to automatically set the <code>compression</code> parameter. 
+      If set to the empty string <code>""</code>, the tuples are read from stdin until pressing <code>Ctrl + D</code>. 
+      This is restricted to one stdin resource per program. If omitted, this is set based on the predicate name, 
+      file format and compression type: <code>&lt;predicate-name&gt;.&lt;format&gt;.&lt;compression&gt;</code>.</> 
+  },
+  format: { 
+    name: <><code>format</code></>, 
+    type: "tuple, single value", 
+    description: <>The input-format of the imported data. 
+      Might be <code>int</code>, <code>double</code>, <code>string</code>, <code>rdf</code> or <code>skip</code>.</> 
+  },
+  compression: { 
+    name: <><code>compression</code></>, 
+    type: "String", 
+    description: <>The compression to use. Currently only <code>gzip</code> or <code>none</code> is supported. 
+      This will normally be guessed correctly from the file extension, 
+      but can be useful for non-standard file names or URLs</> 
+    },
+  limit: { 
+    name: <><code>limit</code></>, 
+    type: "Number", 
+    description: "The maximum number of tuples to export. (great for testing when working with large files)" 
```

Comment:
the component is called `ImportParameters`, this refers to `export`. Ideally, we would use this data for _both_ import and export documentation (which likely requires some modifications to the schema). Otherwise we have the same data duplicated _in two different formats and markup languages_.